### PR TITLE
Make istiod.enabled always true

### PR DESCRIPTION
**Pull request by howardjohn**
This is for master only, not 1.5 backport

### CODE REVIEW

**Review by elfinhe — APPROVED** [review]
now i do see why helm template is hard to maintain...

**Comment by howardjohn** [comment]
> now i do see why helm template is hard to maintain...

Just keep deleting stuff, it gets easier :slightly_smiling_face:

File: manifests/istio-control/istio-config/templates/deployment.yaml [hunk 1]
```diff
@@ -194,16 +171,6 @@ spec:
       securityContext:
         fsGroup: 1337
       volumes:
-  {{- if or .Values.global.controlPlaneSecurityEnabled (and .Values.global.configValidation (not .Values.global.istiod.enabled)) }}
-      - name: istio-certs
```
Comment (beckend) [inline manifests/istio-control/istio-config/templates/deployment.yaml]: removing this is causing an error deploying galley manifest

Comment (howardjohn) [reply manifests/istio-control/istio-config/templates/deployment.yaml]: can you open an issue with more details? it's very likely we will remove galley entirely by the time we ship 1.6 though, and this commit is not included in 1.5

**Pull request merged**